sample/src/main/res/xml/demo_preferences.xml

```diff
@@ -20,6 +20,16 @@
         app:spectrum_colors="@array/demo_colors_expanded" />
 
     <com.thebluealliance.spectrum.SpectrumPreferenceCompat
+        android:defaultValue="@color/md_indigo_500"
+        android:key="demo_preference_6"
+        android:summary="You can make all the colors have the same color border or your choosing"
```

Comment:
Typo, should be "*of* your choosing". Also, can you stick with one title/summary for both the dialog and preference demos?